Client/src/hooks/usePagination.ts

```diff
@@ -0,0 +1,62 @@
+
+
+import { useEffect, useState } from "react";
+import { useAppDispatch } from "./redux";
+import { useNotify } from "./useNotify";
+import { notification } from "antd";
+import { Filters } from "../types/serviceSliceTypes";
+
+interface usePagintaionParams
+{
+    setContent:Function;
+    getContentCB:Function;
+    refetchOnMountOrArgChange?:boolean;
+    filtersParams?:any;
+}
+
+const initialFilters:Filters = {
+    page: 1,
+    limit: 5,
+};
+
```

Comment:
It might be a good idea to add some type annotations to return value of the usePagination hook to improve the readability and the maintainability of the code.